unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/UserContextMenu/Scripts/UserContextMenu.cs

```diff
@@ -274,11 +277,16 @@ private void OnBlockUserButtonPressed()
 
     private void HideIfClickedOutside()
     {
-        if (Input.GetMouseButtonDown(0) &&
-            !RectTransformUtility.RectangleContainsScreenPoint(rectTransform, Input.mousePosition))
+        if (!Input.GetMouseButtonDown(0)) return;
+        var pointerEventData = new PointerEventData(EventSystem.current)
```

Comment:
Approving but this should be changed for performance reasons to: 
`RectTransformUtility.RectangleContainsScreenPoint(rectTransform, Input.mousePosition, DataStore.i.camera.hudsCamera.Get()))`

Reply:
i tried that change, it doesnt work.. i mean the buttons are not triggering the click event because its getting hidden before caused by pointer area calculation issues